code/modules/mob/living/basic/lavaland/mook/mook.dm

```diff
@@ -0,0 +1,405 @@
+	return ..()
+
+/mob/living/basic/mining/mook/worker
+	desc = "This unhealthy looking primitive is wielding a rudimentary hatchet, swinging it with wild abandon. One isn't much of a threat, but in numbers they can quickly overwhelm a superior opponent."
+	gender = MALE
+	melee_damage_lower = 15
+	melee_damage_upper = 15
+	ai_controller = /datum/ai_controller/basic_controller/mook
+	is_healer = FALSE
+
+/mob/living/basic/mining/mook/worker/Initialize(mapload)
+	. = ..()
+	neutral_stance = mutable_appearance(icon, "mook_axe_overlay")
+	attack_stance = mutable_appearance(icon, "axe_strike_overlay")
+	update_appearance()
+	var/datum/action/cooldown/mob_cooldown/mook_ability/mook_leap/leap = new(src)
+	leap.Grant(src)
+	ai_controller.set_blackboard_key(BB_MOOK_LEAP_ABILITY, leap)
+
+/mob/living/basic/mining/mook/worker/attack_sequence(atom/target)
+	. = ..()
+	if(. & COMPONENT_HOSTILE_NO_ATTACK)
+		return
+
+	if(attack_state == MOOK_ATTACK_STRIKE)
+		return COMPONENT_HOSTILE_NO_ATTACK
+
+	change_combatant_state(state = MOOK_ATTACK_STRIKE)
+	addtimer(CALLBACK(src, PROC_REF(change_combatant_state), MOOK_ATTACK_NEUTRAL), 0.3 SECONDS)
+
+/mob/living/basic/mining/mook/worker/update_overlays()
+	. = ..()
+	if(stat == DEAD)
+		return
+
+	switch(attack_state)
+		if(MOOK_ATTACK_STRIKE)
+			. += attack_stance
+		if(MOOK_ATTACK_NEUTRAL)
+			. += neutral_stance
+
+/mob/living/basic/mining/mook/worker/bard
+	desc = "It's holding a guitar?"
+	melee_damage_lower = 10
+	melee_damage_upper = 10
+	gender = MALE
+	attack_sound = 'sound/weapons/stringsmash.ogg'
+	death_sound = 'sound/voice/mook_death.ogg'
+	ai_controller = /datum/ai_controller/basic_controller/mook/bard
+	///our guitar
+	var/obj/item/instrument/guitar/held_guitar
+
+/mob/living/basic/mining/mook/worker/bard/Initialize(mapload)
+	. = ..()
+	neutral_stance = mutable_appearance(icon, "bard_overlay")
+	attack_stance = mutable_appearance(icon, "bard_strike")
+	held_guitar = new(src)
+	ai_controller.set_blackboard_key(BB_SONG_INSTRUMENT, held_guitar)
+	update_appearance()
+
+/mob/living/basic/mining/mook/worker/tribal_chief
+	name = "tribal chief"
+	desc = "Acknowledge him!"
+	gender = MALE
+	melee_damage_lower = 20
+	melee_damage_upper = 20
+	ai_controller = /datum/ai_controller/basic_controller/mook/tribal_chief
+	///overlay in our neutral state
+	var/mutable_appearance/chief_neutral
+	///overlay in our striking state
+	var/mutable_appearance/chief_strike
+	///overlay in our active state
+	var/mutable_appearance/chief_active
+	///overlay in our warmup state
+	var/mutable_appearance/chief_warmup
+
+/mob/living/basic/mining/mook/worker/tribal_chief/Initialize(mapload)
+	. = ..()
+	chief_neutral = mutable_appearance(icon, "mook_chief")
+	chief_strike = mutable_appearance(icon, "mook_chief_strike")
+	chief_active = mutable_appearance(icon, "mook_chief_leap")
+	chief_warmup = mutable_appearance(icon, "mook_chief_warmup")
```

Comment:
nah we can do it like this

```suggestion
	///overlay in our neutral state
	var/static/mutable_appearance/chief_neutral = mutable_appearance(icon, "mook_chief")
	///overlay in our striking state
	var/static/mutable_appearance/chief_strike = mutable_appearance(icon, "mook_chief_strike")
	///overlay in our active state
	var/static/mutable_appearance/chief_active = mutable_appearance(icon, "mook_chief_leap")
	///overlay in our warmup state
	var/static/mutable_appearance/chief_warmup = mutable_appearance(icon, "mook_chief_warmup")

/mob/living/basic/mining/mook/worker/tribal_chief/Initialize(mapload)
	. = ..()
```

tada. if you don't believe this works look at `/datum/component/wet_floor`.

Reply:
i tried this and the integration test cr*pped itself